modules/ztp-precaching-downloading-artifacts.adoc

```diff
@@ -38,19 +38,19 @@ To remove this restriction, you can precede your commands with `taskset 0xffffff
 [id="ztp-preparing-ocp-images_{context}"]
 == Preparing to download the {product-title} images
 
-To download {product-title} container images, you need to know the multicluster engine (MCE) version. When you use the `--du-profile` flag, you also need to specify the {rh-rhacm-first} version running in the hub cluster that is going to provision the {sno}.
+To download {product-title} container images, you need to know the {mce-short} version. When you use the `--du-profile` flag, you also need to specify the {rh-rhacm-first} version running in the hub cluster that is going to provision the {sno}.
 
 .Prerequisites
 
-* You have {rh-rhacm} and MCE installed.
+* You have {rh-rhacm} and {mce-short} installed.
 * You partitioned the storage device.
 * You have enough space for the images on the partitioned device.
 * You connected the bare-metal server to the Internet.
 * You have a valid pull secret.
 
 .Procedure
 
-. Check the {rh-rhacm} and MCE version by running the following commands in the hub cluster:
+. Check the {rh-rhacm} and {mce-short} version by running the following commands in the hub cluster:
 +
```

Comment:
Again, is "multicluster engine operator version" the same as "multicluster engine version"?

I would also change "version" to "versions" as these appear to be different artifacts so they might have different versions.

Reply:
I would go with "the {rh-rhacm} version and the multicluster engine version" but it's your call

Reply:
Done